### ISSUE TITLE
[Proposal] Change default hostname to "openhab"

### ISSUE THREAD

**Issue by kubawolanin** [openhab/openhabian#177]
I'm aware that there's an option to easily change the hostname by yourself, although it might be cool for the newcomers to just type `openhab:8080` (12 chars) instead `openhabianpi:8080` (17 chars) 😉 

My personal preference is `home`. Combined with my [local-alias](https://kubawolanin.github.io/local-alias/) it's pretty handy:
![](https://kubawolanin.github.io/images/local-alias.gif)

**Comment by ThomDietrich** [comment]
Hey Kuba,
not sure how I feel about this change. openhab is the name if the software, openHABian is the name of the distribution, openHABianPi the default name for an openHABian empowered Pi. I'd find it irritating to name a device "openhab".
The user can of course do that and a function to change the hostname is available via `sudo openhabian-config`.

Wdyt?

**Comment by kubawolanin** [comment]
Sure! Good reasoning. 
Closing the issue then :-)

**Comment by ThomDietrich** [comment]
Didn't want to shut you off. I'm simply not sure :) Are you still pro "openhab"?

**Comment by kubawolanin** [comment]
Well, my point was that the URL it quite too long for me. Just a humble opinion ;)

And since we're hosting openHAB on our machine (that's dedicated to openHAB specifically), why not make it easier?

Take [Volumio](https://github.com/volumio/Volumio2) as an example. It's a open-source software that transforms your RPi into an audio device. The idea is pretty much aligned with how openHABian operates - you flash an image and you're ready to go.
Volumio's UI is accessible from http://volumio.local/

I think it would also make sense to add some (optional?) script that would route `8080` to `80`, so you don't need to type port at all:
```
iptables -t nat -A PREROUTING -p tcp -d openhabian.IP.address --dport 80 -j DNAT --to-destination openhabian.IP.address:8080
```

But that's a subject for another issue :)

**Comment by ThomDietrich** [comment]
Hmm let me think about it

Regarding the port topic: That's an interesting issue. The non-responsive port 80 is probably one of the first hurdles for new users.
We would also need to consider the optional nginx reverse proxy (with auth and https). I'm not so sure about the transparent rerouting. How would you think about a redirect from 80 to 8080? I could implement this as a simple nginx config, which would then be replaced as soon as the reverse proxy setup is performed by the user.

**Comment by jscottnz** [comment]
i prefer `openhab` as that is the software - I don't care about the implementation - just that i can use it.

I actually have 3 openhab instances:
openhab1.local : rpi openhabian
openhab2.local : rpi openhabian
and an unnamed osx box

**Comment by ThomDietrich** [comment]
I'm not particularly against this.

@EliasGabrielsson @mues-lee @BClark09 ?

**Comment by mues-lee** [comment]
i don't really have a opinion on that, there are some users on the forum that use openHAB and openHABian synonymous anyways

**Comment by ThomDietrich** [comment]
Well that can't be good 😊

**Comment by EliasGabrielsson** [comment]
I am positive to use `openhab` as default url, use it myself at home. Compared with the common pattern `ftp.domain.com` it kind of make sense to have the service type within the domain name.

**Comment by ThomDietrich** [comment]
Okay then. Anyone free to open a PR

**Comment by kubawolanin** [comment]
> Okay then. Anyone feel free to open a PR

I can take it 🙂 

One advantage of having unified hostname would be unified URL reference in docs, community forum etc.
So instead of writing `http://<yourip>:8080/basicui/` we would have `http://openhab:8080/basicui`, regardless of the platform.

**Comment by kaikreuzer** [comment]
I am all for it 👍

**Comment by ThomDietrich** [comment]
[**@kubawolanin**](https://github.com/kubawolanin) commented on [28. Mai 2018, 14:27 MESZ](https://github.com/openhab/openhabian/issues/177#issuecomment-392513906 "2018-05-28T12:27:24Z - Replied by Github Reply Comments"):
> I can take it 🙂 
> 
> ...unified URL...

Thanks! You are right, that is a nice benefit!

**Comment by EliasGabrielsson** [comment]
Implemented in openHABian 1.5
